src/crawlee/browsers/_playwright_browser_controller.py

```diff
@@ -130,17 +129,20 @@ def _on_page_close(self, page: Page) -> None:
         """Handle actions after a page is closed."""
         self._pages.remove(page)
 
-    async def _create_browser_context(self, proxy_info: ProxyInfo | None = None) -> BrowserContext:
+    async def _create_browser_context(
+        self, page_options: Mapping[str, Any] | None = None, proxy_info: ProxyInfo | None = None
+    ) -> BrowserContext:
         """Create a new browser context with the specified proxy settings."""
         if self._header_generator:
             common_headers = self._header_generator.get_common_headers()
             sec_ch_ua_headers = self._header_generator.get_sec_ch_ua_headers(browser_type=self.browser_type)
             user_agent_header = self._header_generator.get_user_agent_header(browser_type=self.browser_type)
             extra_http_headers = dict(common_headers | sec_ch_ua_headers | user_agent_header)
-            user_agent = user_agent_header.get('User-Agent')
```

Comment:
This was here to replace the default "headless"-something user agent. Why is it removed? Does it mean we're now using the default (headless) user agent?

Reply:
We have a similar test

https://github.com/apify/crawlee-python/blob/master/tests/unit/playwright_crawler/test_playwright_crawler.py#L106